libvast/src/layout_version.cpp

```diff
@@ -0,0 +1,74 @@
+#include "vast/layout_version.hpp"
+
+#include "vast/detail/assert.hpp"
+#include "vast/error.hpp"
+#include "vast/io/read.hpp"
+#include "vast/io/write.hpp"
+#include "vast/logger.hpp"
+
+#include <fstream>
+#include <iterator>
+
+#include "caf/expected.hpp"
+
+namespace {
```

Comment:
Can you move this anonymous namespace inside the `vast` namespace such that you don't need to specify `vast::` everywhere?